### ISSUE TITLE
Resolution too small .. 

### ISSUE THREAD

**Issue by zsz00** [IanButterworth/VideoInTerminal.jl#4]
I try this player, find resolution too small,  not clear.
if use Sixel encoder, it will show clear. 

![image](https://user-images.githubusercontent.com/12711073/179665660-0f4acb1c-2d7c-49e1-bed6-7d647e6a1d5e.png)

**Comment by IanButterworth** [comment]
Development on this package stopped because it was due to be merged into ImageView.jl / ImageInTerminal.jl, which does have sixel support if I understand correctly

@johnnychen94 is that still the idea?

**Comment by johnnychen94** [comment]
Yes. ImageInTerminal also needs heavy refactoring to separate the core encoding part apart so that other packages can reuse its functionality without worrying about `Base.show` method getting overridden. Such refactoring is happening to https://github.com/JuliaImages/XTermColors.jl.

It's a bit of low priority to me given my limited time in contributing to Julia -- it will slowly move forward, but I can't give a guarantee about when it will be done.

---

If you want sixel supported (in this package), you can directly use https://github.com/JuliaIO/Sixel.jl. I tried video encoding when I initially wrote Sixel.jl and it worked somehow, but it's not easy work to align the sixel sequence together with normal ascii control sequence. See also the discussion in https://github.com/FedeClaudi/Term.jl/discussions/94
